python/packages/autogen-core/docs/src/user-guide/agentchat-user-guide/magentic-one.md

```diff
@@ -122,13 +145,15 @@ While the default multimodal LLM we use for all agents is GPT-4o, Magentic-One i
 ## Citation
 
 ```
+
 @misc{fourney2024magenticonegeneralistmultiagentsolving,
       title={Magentic-One: A Generalist Multi-Agent System for Solving Complex Tasks},
       author={Adam Fourney and Gagan Bansal and Hussein Mozannar and Cheng Tan and Eduardo Salinas and Erkang and Zhu and Friederike Niedtner and Grace Proebsting and Griffin Bassman and Jack Gerrits and Jacob Alber and Peter Chang and Ricky Loynd and Robert West and Victor Dibia and Ahmed Awadallah and Ece Kamar and Rafah Hosn and Saleema Amershi},
       year={2024},
       eprint={2411.04468},
       archivePrefix={arXiv},
       primaryClass={cs.AI},
-      url={https://arxiv.org/abs/2411.04468},
+      url={<https://arxiv.org/abs/2411.04468}>,
```

Comment:
this looks off with the {<}> - is that the way its meant to be?

Reply:
No, it was a typo.